src/DeviceListener.js

```diff
@@ -147,6 +156,20 @@ export default class DeviceListener {
                 }
             }
             return;
+        } else if (await cli.secretStorageKeyNeedsUpgrade()) {
+            if (this._dismissedThisDeviceToast) {
+                ToastStore.sharedInstance().dismissToast(THIS_DEVICE_TOAST_KEY);
+                return;
+            }
+
+            // FIXME: do we a different message?
```

Comment:
I think it's probably okay to reuse the same...?

Reply:
Yeah, I think it's probably fine.